src/edfi-performance-test/edfi_performance_test/factories/resources/education.py

```diff
@@ -66,8 +66,10 @@ class LocalEducationAgencyFactory(APIFactory):
 
 class EducationContentFactory(APIFactory):
     contentIdentifier = UniqueIdAttribute(num_chars=60)
-    description = "A learning resource for all grade levels"
+    shortDescription = "A learning resource for all grade levels"
     namespace = "uri://ed-fi.org"
+    learningResourceMetadataURI = "uri://ed-fi.org/whatever"
+    contentClassDescriptor = "uri://ed-fi.org/ContentClassDescriptor#Presentation"
```

Comment:
Dealing with the known problem of a required Choice: ODS/API does not enforce, and Meadowlark does. I have proposed that we standardize on what the ODS/API does, which means changing Meadowlark. In the meantime, these extra fields at least let the processing continue without error against Meadowlark.